test/libsolidity/syntaxTests/nameAndTypeResolution/428_bare_revert.sol

```diff
@@ -5,4 +5,4 @@ contract C {
     }
 }
 // ----
-// TypeError 2144: (81-87): No matching declaration found after variable lookup.
+// Warning 6133: (81-87): Statement has no effect.
```

Comment:
Maybe it would be better to move `nameAndTypeResolution/428_bare_revert.sol` and `nameAndTypeResolution/432_bare_require.sol` to `errors/`?